main.js

```diff
@@ -0,0 +1,111 @@
+const game = {
+    board: [[5, 4, 3, 2, 1], [], []],
+    peg1EmptiedOnce: false,  // Track whether peg1 has been emptied
+
+    // Display the current state of the board using map
+    displayBoard() {
+        console.log("--- Board State ---");
+        this.board.map(peg => {
```

Comment:
why are you using map? this is the wrong helper, map is less performant as returns a new array with elements, you wanted to use here forEach